### PR TITLE
docs: update tutorial po

### PR DESCRIPTION
**Pull request by hengzi**
更新”水平联邦：图像分类“、“使用Pytorch后端来进行联邦学习”和”水平联邦XGBoost“三篇教程的文档。

### CODE REVIEW

**Comment by Candicepan** [comment]
remember to sign the CLA follow the steps above

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/Federated_Xgboost.po [hunk 1]
```diff
@@ -28,10 +28,11 @@ msgid ""
 msgstr "以下代码仅作为演示用，请勿直接在生产环境使用。"
 
 #: ../../tutorial/Federated_Xgboost.ipynb:31
+#, fuzzy
```
Comment (zhouaihui) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/Federated_Xgboost.po:31]: Remove `fuzzy`

**Comment by hengzi** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/locales/zh_CN/LC_MESSAGES/tutorial/Federate_Learning_for_Image_Classification.po [hunk 2]
```diff
@@ -109,109 +109,114 @@ msgid ""
...
-"Bob’s data is combined, a model with an accuracy close to 0.995 can be "
-"obtained. In addition, the generalization performance of the model "
-"jointly trained with multi-party data will also be better"
-msgstr ""
-"上面两个实验模拟了一个典型的水平联邦场景的训练问题，\\* alice和bob拥有类型的图片\\* "
-"每一方只有样本的一部分数据，但是双方的训练目的是一致的如果alice只用自己的一方数据来训练模型，能够得到一个精确度0.945的模型，但是如果联合bob的数据之后，可以获得一个精确度接近0.995的模型，而且多方数据联合训练的模型的泛化性能也会更好"
-
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:850
+"horizontal federation scenario, Alice and Bob have same type of data. "
+"Each side had only a portion of the sample, but the training objectives "
```
Comment (zhangxingmeng) [inline docs/locales/zh_CN/LC_MESSAGES/tutorial/Federate_Learning_for_Image_Classification.po:194]: Here is a display issue， the two lines with "*" are intended to be displayed as list. I will fix this problem later. thanks for your work

**Review by zhangxingmeng — APPROVED** [review]
LGTM

**Pull request merged**